pkg/cmd/pulumi/show/show.go

```diff
@@ -0,0 +1,117 @@
+package show
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pulumi/pulumi/pkg/v3/backend/display"
+	"github.com/pulumi/pulumi/pkg/v3/backend/secrets"
+	"github.com/pulumi/pulumi/pkg/v3/cmd/pulumi/backend"
+	"github.com/pulumi/pulumi/pkg/v3/cmd/pulumi/stack"
+	"github.com/pulumi/pulumi/pkg/v3/workspace"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func ShowCmd() *cobra.Command {
+	var stackName string
+	var rf resourceFilters
+
+	cmd := &cobra.Command{
+		Use:   "show",
+		Short: "show resources in the stack",
+		Long:  "show resources in the  stack",
```

Comment:
why the double space if you just copy pasted from above?  Also take a look at other commands for consistency here, e.g. first letter is upper case, long description should be an actual description of what the command does.